Index: dns/Makefile
===================================================================
--- dns/Makefile	(revision 520519)
+++ dns/Makefile	(working copy)
@@ -110,6 +110,7 @@
     SUBDIR += mydns
     SUBDIR += mydns-ng
     SUBDIR += namesilo_ddns
+    SUBDIR += nextdns
     SUBDIR += noip
     SUBDIR += nsd
     SUBDIR += nsec3walker
Index: dns/nextdns/Makefile
===================================================================
--- dns/nextdns/Makefile	(nonexistent)
+++ dns/nextdns/Makefile	(working copy)
@@ -0,0 +1,34 @@
+# $FreeBSD$
+
+PORTNAME=	nextdns
+PORTVERSION=	1.3.4
+CATEGORIES=	dns security
+
+MAINTAINER=	rs@nextdns.io
+COMMENT=	NextDNS DNS53 to DNS over HTTPS proxy
+
+LICENSE=	MIT
+LICENSE_FILE=	${WRKSRC}/LICENSE
+
+RUN_DEPENDS=	ca_root_nss>=3.35:security/ca_root_nss
+
+USES=		go:modules
+PLIST_FILES=	sbin/nextdns
+USE_GITHUB=	yes
+GO_TARGET=	.:${PREFIX}/sbin/nextdns
+GH_ACCOUNT=	nextdns
+GH_TAGNAME=	v1.3.3
+GH_TUPLE=	\
+		cenkalti:backoff:v2.2.1:cenkalti_backoff/vendor/github.com/cenkalti/backoff \
+		cespare:xxhash:v1.1.0:cespare_xxhash/vendor/github.com/cespare/xxhash \
+		denisbrodbeck:machineid:v1.0.1:denisbrodbeck_machineid/vendor/github.com/denisbrodbeck/machineid \
+		golang:crypto:9ee001bba392:golang_crypto/vendor/golang.org/x/crypto \
+		golang:net:a882066a44e0:golang_net/vendor/golang.org/x/net \
+		golang:sys:52ab43148777:golang_sys/vendor/golang.org/x/sys \
+		grandcat:zeroconf:85eadb44205c:grandcat_zeroconf/vendor/github.com/grandcat/zeroconf \
+		miekg:dns:v1.1.22:miekg_dns/vendor/github.com/miekg/dns \
+		rs:service:b1a37fd90075:rs_service/vendor/github.com/kardianos/service
+
+SUB_FILES=	pkg-message
+
+.include <bsd.port.mk>

Property changes on: dns/nextdns/Makefile
___________________________________________________________________
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:keywords
## -0,0 +1 ##
+FreeBSD=%H
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Index: dns/nextdns/distinfo
===================================================================
--- dns/nextdns/distinfo	(nonexistent)
+++ dns/nextdns/distinfo	(working copy)
@@ -0,0 +1,21 @@
+TIMESTAMP = 1576771190
+SHA256 (nextdns-nextdns-1.3.4-v1.3.3_GH0.tar.gz) = d3d7a00ba5065f0a4bf1cb982dd6d7c67cdef750fa841bedc3c601434353d47b
+SIZE (nextdns-nextdns-1.3.4-v1.3.3_GH0.tar.gz) = 33758
+SHA256 (cenkalti-backoff-v2.2.1_GH0.tar.gz) = a2c29d0184e7afc415975cf2689723028d2686ffbb67fe0999ab1d691e6d16db
+SIZE (cenkalti-backoff-v2.2.1_GH0.tar.gz) = 8623
+SHA256 (cespare-xxhash-v1.1.0_GH0.tar.gz) = 9418be390574092f0ca989b9ae2f5450270ead8125a635a100850b28c8c9c495
+SIZE (cespare-xxhash-v1.1.0_GH0.tar.gz) = 8200
+SHA256 (denisbrodbeck-machineid-v1.0.1_GH0.tar.gz) = 0a01d2b131419cb2b9f7d910a2c90b824449bd91005b47efb235dc707665337a
+SIZE (denisbrodbeck-machineid-v1.0.1_GH0.tar.gz) = 27807
+SHA256 (golang-crypto-9ee001bba392_GH0.tar.gz) = a33baa9b2a1eff49d95fb0ec97dbd66c703b9ae9808215ece254d6445a751b48
+SIZE (golang-crypto-9ee001bba392_GH0.tar.gz) = 1693135
+SHA256 (golang-net-a882066a44e0_GH0.tar.gz) = cf3c88a5e37296bafab87cac0dc1d4f5272ef7a9726d7b57dd78171b2ac63b4c
+SIZE (golang-net-a882066a44e0_GH0.tar.gz) = 1165247
+SHA256 (golang-sys-52ab43148777_GH0.tar.gz) = 8b71b7fcc21c66ffdc66affd45a489fadb2d15b674993a10f72606fd303b385d
+SIZE (golang-sys-52ab43148777_GH0.tar.gz) = 1515147
+SHA256 (grandcat-zeroconf-85eadb44205c_GH0.tar.gz) = ba43a3a468bea28d4217be4b463bbf6a338ad39ae0280cecc4dd8cd5e4129b59
+SIZE (grandcat-zeroconf-85eadb44205c_GH0.tar.gz) = 14431
+SHA256 (miekg-dns-v1.1.22_GH0.tar.gz) = 32446249a35d682913bc0d56a1126c0aec8712d3714c5b5ad078f14b682ba0ef
+SIZE (miekg-dns-v1.1.22_GH0.tar.gz) = 178978
+SHA256 (rs-service-b1a37fd90075_GH0.tar.gz) = de4a8390ce0ee9249cb9d2705f1a046d45bfc49e2981e26888cfa2ed19d4ffcd
+SIZE (rs-service-b1a37fd90075_GH0.tar.gz) = 23873

Property changes on: dns/nextdns/distinfo
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Index: dns/nextdns/files/pkg-message.in
===================================================================
--- dns/nextdns/files/pkg-message.in	(nonexistent)
+++ dns/nextdns/files/pkg-message.in	(working copy)
@@ -0,0 +1,20 @@
+[
+{
+  type: install
+  message: <<EOM
+To configure NextDNS, run the following command as root:
+
+    nextdns install -config <your_conf_id>
+
+By default, nextdns listen on localhost only. To setup on a server,
+add the -listen :53 option in order to listen on all interfaces.
+
+Go on https://nextdns.io to create a custom configuration ID.
+
+Then setup your DNS manually to 127.0.0.1 or by running as root:
+
+    nextdns activate
+
+EOM
+}
+]

Property changes on: dns/nextdns/files/pkg-message.in
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Index: dns/nextdns/pkg-deinstall
===================================================================
--- dns/nextdns/pkg-deinstall	(nonexistent)
+++ dns/nextdns/pkg-deinstall	(working copy)
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ x"$2" != x"DEINSTALL" ]; then
+    exit 0
+fi
+
+${PKG_PREFIX}/nextdns uninstall >/dev/null 2>/dev/null
+rm -f ${PKG_PREFIX}/etc/nextdns.conf
+
+exit 0

Property changes on: dns/nextdns/pkg-deinstall
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
Index: dns/nextdns/pkg-descr
===================================================================
--- dns/nextdns/pkg-descr	(nonexistent)
+++ dns/nextdns/pkg-descr	(working copy)
@@ -0,0 +1,3 @@
+NextDNS DNS53 to DNS over HTTPS proxy
+
+WWW: https://github.com/nextdns/nextdns

Property changes on: dns/nextdns/pkg-descr
___________________________________________________________________
Added: fbsd:nokeywords
## -0,0 +1 ##
+yes
\ No newline at end of property
Added: svn:eol-style
## -0,0 +1 ##
+native
\ No newline at end of property
Added: svn:mime-type
## -0,0 +1 ##
+text/plain
\ No newline at end of property
